index.js

```diff
@@ -70,6 +70,12 @@ const envconst = {
      automatically granted CI access
    */
   CI_USER_WHITELIST: '', // comma separated, no spaces
+
+  /*
+     Labels that are specific to instance of CI gate.
+     These labels are passed on to buildkite.
+   */
+  CUSTOM_LABELS: 'testnet,', // comma separated, no spaces
```

Comment:
`CUSTOM_LABELS` should be the empty string for the default configuration

Reply:
How about `CUSTOM_PR_LABEL_BUILDS`?

Reply:
Sounds good.